src/angular/alert/alert.html

```diff
@@ -16,10 +16,6 @@
     (click)="_handleDismiss($event)"
   >
     <sbb-icon svgIcon="kom:cross-small"></sbb-icon>
-    <span
-      i18n="Hidden button label to close the alert@@sbbAlertCloseAlert"
-      class="cdk-visually-hidden"
-      >Close message</span
-    >
+    <span class="cdk-visually-hidden">{{_labelClose}}</span>
```

Comment:
Doesn't prettier format this as `{{ _labelClose }}`?

Reply:
it seems that prettier ignores this (I can write whatever i want and nothing changes). But I fixed it manually